src/chord.cpp

```diff
@@ -817,4 +818,61 @@ int Chord::ResetDrawing(FunctorParams *functorParams)
     return FUNCTOR_CONTINUE;
 }
 
+int Chord::JustifyY(FunctorParams *functorParams)
+{
+    JustifyYParams *params = vrv_params_cast<JustifyYParams *>(functorParams);
+    assert(params);
+    assert(params->m_justificationSum > 0);
+
+    // Check if chord spreads across several staves
+    std::list<Staff *> extremalStaves;
+    for (Note *note : { this->GetTopNote(), this->GetBottomNote() }) {
+        Layer *layer = NULL;
+        Staff *staff = note->GetCrossStaff(layer);
+        if (!staff) staff = vrv_cast<Staff *>(this->GetFirstAncestor(STAFF));
+        assert(staff);
+        extremalStaves.push_back(staff);
+    }
+    assert(extremalStaves.size() == 2);
```

Comment:
In theory we should not have chords with a single note, but I think it can happen. Wouldn't this make it crash?

Reply:
- Both `GetTopNote()` and `GetBottomNote()` assert to return a note. I assume it would be the same note, if the chord consists just of one note. Then `extremalStaves` still consists of two (equal) entries.
- The loop runs through a container of fixed size 2, each time pushing a staff to a list. So afterwards the list size should be 2 as well.

Reply:
Just tested it with a chord consisting of a single note. It is how I described above, no problem.

Reply:
Yes, you are right! Sorry